tests/.coveragerc

```diff
@@ -0,0 +1,17 @@
+# .coveragerc to control coverage.py
+[run]
+# don't report on coverage of files in the tests dir itself
+omit = 
+# easier68k/tests/*
+  */__init__.py
```

Comment:
Looks like `__init__.py` files are still included in the coverage results in Travis. Might be missing a comma?

Reply:
No, actually that's the correct syntax (http://coverage.readthedocs.io/en/coverage-4.5.1/config.html#run)

`[report] omit` might be what we are looking for though. http://coverage.readthedocs.io/en/coverage-4.5.1/config.html#report